src/models/Event.js

```diff
@@ -68,15 +69,11 @@ const eventSchema = new Schema({
         minlength: [2, "Category must be minimum 2 characters long!"],
         maxlength: [30, "Category must be maximum 30 characters long!"],
     },
-    
+
     likes: [{ type: ObjectId, ref: 'User' } || { type: ObjectId, ref: 'Organization' }],
     // TODO: We must add Admin model later!
     creator: { type: ObjectId, ref: 'Admin', required: true } || { type: ObjectId, ref: 'Organization', required: true },
-    winners: [
-        { name: String, vehicle: String },
-        { name: String, vehicle: String },
-        { name: String, vehicle: String }
-    ],
+    winners: { firstPlace: { name: String, vehicle: String } },
```

Comment:
No, just keep it an array with objects with properties name, vehicle and place.

Reply:
Yes this is good approach, just objects and inside will be the place.